docs/man/docker.1.md

```diff
@@ -292,79 +292,192 @@ inside it)
 
 # STORAGE DRIVER OPTIONS
 
-Options to storage backend can be specified with **--storage-opt** flags. The
-only backend which currently takes options is *devicemapper*. Therefore use these
+Docker uses storage backends (or "graphdriver" in Docker internals) to create
```

Comment:
the (or ... ) seems a little weird. Perhaps:

(known as graph drivers in the Docker internals)